build.ps1

```diff
@@ -409,6 +412,10 @@ End {
             $extraParams.Add('Tags', $tags)
         }
 
+        if((Get-Module -ListAvailable pester -ErrorAction Ignore -Force) -or !$SkipPesterInstall.IsPresent)
```

Comment:
still need to rename the switch... the name says skip, but the action is to force.

Reply:
Fixed